django_netjsonconfig/base/base.py

```diff
@@ -135,15 +135,21 @@ def get_backend_instance(self, template_instances=None):
         """
         backend = self.backend_class
         kwargs = {'config': self.get_config()}
+        default_values = {}
         # determine if we can pass templates
         # expecting a many2many relationship
         if hasattr(self, 'templates'):
             if template_instances is None:
                 template_instances = self.templates.all()
-            kwargs['templates'] = [t.config for t in template_instances]
+            templates_list = list()
+            for t in template_instances:
+                templates_list.append(t.config)
+                default_values.update(t.default_values)
```

Comment:
here we have to use the `get_context` method we added to `Template`, otherwise that method is useless.
Moreover, this way the code is more generic and loosely coupled.

So:

```python
context.update(t.get_context())
```